src/LaunchDarkly.EventSource/EventSource.cs

```diff
@@ -180,10 +228,13 @@ public async Task StartAsync()
         /// </summary>
         public void Close()
         {
-            if (ReadyState == ReadyState.Raw || ReadyState == ReadyState.Shutdown) return;
```

Comment:
Removed this `return` because we do want to make sure we dispose of the HTTP client even if we're not currently active.